### ISSUE TITLE
Menu and Dropdown

### ISSUE THREAD

**Issue by ahmedlhanafy** [callstack/react-native-paper#9]
[MD Docs Link](https://material.google.com/components/menus.html#menus-usage)

**Comment by marktkimball** [comment]
Is anything happening with this component?  I love this component library, but without the drop down menu to use with the "More" icon in a Toolbar, it seems slightly gimped.  What are people doing to display a "more menu" from the Toolbar?

**Comment by Trancever** [comment]
@marktkimball @kpsroka is currently working on that component, so it should be released in the near future.

**Comment by satya164** [comment]
If you're on Android, you can currently use the undocumented popup menu in React Native:

```js
UIManager.showPopupMenu(
  ReactNative.findNodeHandle(
    this._anchor // Ref to the item that'll act as anchor to the popup menu
  ),
  ['One', 'Two', 'Three'],
  error => { /* handle error */ },
  (
    action, // "itemSelected", "dismissed"
    index // index of item that's selected
  ) => { /* handle action /* },
);
```

**Comment by marsonmao** [comment]
Any progress on this component?

**Comment by ferrannp** [comment]
@marsonmao not yet.

**Comment by ahce** [comment]
This [project](https://github.com/n4kz/react-native-material-dropdown) is a reference for this

**Comment by cltsang** [comment]
It's being implemented in #485

**Comment by mgunther** [comment]
Look to react-native-material-menu...
